classes/app.rb

```diff
@@ -61,15 +53,10 @@ def create_book
 
   def create_rental
     puts 'Select a book from the following list by number'
-    @books.each_with_index do |book, index|
-      puts "#{index}) Title: \"#{book.title}\", Author: #{book.author}"
-    end
+    list_books
```

Comment:
- You have done great implementing the suggestions from the previous reviewer, but there are a few changes you need to make before moving to the next milestone. :+1: :100: 

- [ ] Looking at the screenshot below, you can see that I got an error when trying to create a rental, You will need to fix this as your app is meant to still function correctly after the refactoring. :smile: :small_airplane: :wave: 

- - Screenshot reference:
![Selection_031](https://github.com/siddghosh108/school-library/assets/107632780/7d6aa3e9-c435-4574-bdc7-a1ab82eb3e04)